setup.py

```diff
@@ -0,0 +1,63 @@
+import os
+from setuptools import setup
+
+setup(
+    name="wow_addon_updater",
+    version="v1.7.1",
+    description=(
+        "This utility provides an alternative to the Twitch/Curse "
+        "client for management and updating of addons for World of Warcraft. "
+    ),
+    author="Garrett Edwards",
+    url="https://github.com/grrttedwards/wow-addon-manager",
```

Comment:
```suggestion
    url="https://github.com/grrttedwards/wow-addon-manager",
    license="GPLv3",
    classifiers=[
        "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
        "Programming Language :: Python :: 3 :: Only",
        "Operating System :: OS Independent",
    ],
```

I built the ... package? wheel? egg? whatever it's called 😄 and noticed a few little holes in the metadata. There's also a
```
Description: UNKNOWN
Platform: UNKNOWN
```
that goes into the `PKG-INFO` which we may want to fill in.